lib/qesapdeployment.pm

```diff
@@ -423,8 +426,9 @@ sub qesap_prepare_env {
 
     record_info("QESAP conf", "Generating tfvars file");
     push(@log_files, $paths{terraform_dir} . '/' . $provider . "/terraform.tfvars");
-    qesap_execute(cmd => 'configure', verbose => 1);
+    my $exec_rc = qesap_execute(cmd => 'configure', verbose => 1);
```

Comment:
I propose not to change API for this, I propose to directly manage exec_rc within qesap_prepare_env.
Do we have a use case to allowing configuration error?

Reply:
Just to have uniform behavior for all commands and ability to handle the error by calling module instead of library. 
But I don't need that one at least atm. 
Will make it fail directly in the lib.